non-free.md

```diff
@@ -106,3 +107,8 @@
   
 ## Ticketing
   * [SupportPal](https://www.supportpal.com/) `⊘ Proprietary` - Powerful help desk software - easy, fast and intuitive. ([Demo](http://demo.supportpal.com/)) `PHP`
+
+
+## Remote Support
+
+ * [ScreenConnect](https://www.screenconnect.com/) `⊘ Proprietary` - ScreenConnect offers lightning-fast remote support and remote access to connect instantly and solve problems faster.
```

Comment:
There is a self-hosted section on the [pricing page](https://www.screenconnect.com/Pricing).